tests/test_resource_warning.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import time
+from datetime import datetime
+
+import aiohttp
+
+
+async def _resource_warning(recwarn):
+    timeout = 3 * 60
+
+    request_interval = 60
+    request_increase = 30
+    next_request = 0
+
+    async with aiohttp.ClientSession() as client:
+
+        end = time.time() + timeout
+        while True:
+            now = time.time()
+            if now > end:
+                break
+
+            if now > next_request:
+                async with client.get('https://heise.de') as resp:
```

Comment:
```suggestion
                async with client.get(url) as resp:
```
(you can add this suggestion to batch ☝️ and commit two suggestions together)

Reply:
A - too late. Sorry - it's pretty new for me.